course/pages/dagster-essentials/lesson-7/coding-practice-weekly-update-schedule.md

```diff
@@ -23,7 +23,7 @@ The schedule you built should look similar to the following code. Click **View a
 ```python {% obfuscated="true" %}
 import dagster as dg
 
-from ..jobs import weekly_update_job
+from .jobs import weekly_update_job
```

Comment:
Side note, but I tend to prefer absolute imports over relative for this reason.

Reply:
Agreed. I can make a ticket for this sprint to change them in all of Dagster Essentials and Dagster & dbt. Testing with Dagster doesn't use relative paths

Reply:
Sounds good!